src/app/session-lobby/session-lobby.component.ts

```diff
@@ -8,14 +10,21 @@ import { TeacherSessionService } from '../teacher-session.service';
 })
 export class SessionLobbyComponent implements OnInit {
 
-  sessionID: string;
 
-  constructor(public teacherSessionService: TeacherSessionService) {
-    this.sessionID = 'demo-session'; // Temporary until a way to get the session is implemented
-  }
+  sessionID = 'demo-session'; // Temporary until a way to get the session is implemented
+  roundData = {flowerSpeciesIds: ['asclepias_syriaca', 'coreopsis_palmata'],
+                        status: 'fine',
+                        running: false,
+                        currentTime: 0, };
```

Comment:
```suggestion
                        currentTime: teacherRoundService.startTime.time, };
```

We might want to use the demo start time in the demo round data, instead of just putting in `0`. (Credit to @floogulinc for catching this!)